testsuite/classlibrary/TestDocument.sc

```diff
@@ -5,14 +5,15 @@ TestDocument : UnitTest {
 
 	test_new_document_runs_initAction {
 		var success = false, doc, save;
```

Comment:
The `success` variable isn't needed anymore, and it shouldn't be used on line 21 in the assert.